ts/components/portfolio/CreditCardComponent.tsx

```diff
@@ -0,0 +1,141 @@
+                {("\u25cf".repeat(4) + " ").repeat(3) + item.number.slice(-4)}
+              </Text>
+            </Col>
+            <Col size={1}>
+              <Icon
+                type = "MaterialIcons"
+                active={false}
+                style={CreditCardStyle.iconStyle}
+                name="star-border"
+              />
+            </Col>
+            <Col size={1}>
+              <Icon
+                type = "MaterialIcons"
+                active={true}
+                style={CreditCardStyle.iconStyle}
+                name="more-vert"
+              />
+            </Col>
+          </Row>
+          <Row size={1} style = {CreditCardStyle.rowStyle}>
+            <Col>
+              <Text style = { [CreditCardStyle.textStyle, CreditCardStyle.smallTextStyle] } >
+                {I18n.t('creditCardComponent.validUntil') + item.expires}
+              </Text>
+            </Col>
+          </Row>
+          <Row size={2} style = {[CreditCardStyle.rowStyle, CreditCardStyle.whiteBarStyle]}>
+            <Col size={7}>
+              <Text style = { CreditCardStyle.textStyle }>
+                {item.owner.toUpperCase()}
+              </Text>
+            </Col>
+            <Col size={2}>
+              <Image
+                style = {CreditCardStyle.issuerLogo}
+                source= { this.getCreditCardImage(item.number) }/>
+            </Col>
+            <Col size={1}><Text>&nbsp;</Text></Col>
+          </Row>
+          {/* onPress supported by row, but not recognized by typescript
+            * (https://github.com/GeekyAnts/react-native-easy-grid/blob/master/Components/Row.js#L29)
+            */}
+          <Row
+            style = {CreditCardStyle.rowStyle}
+            size={2}
+            {...{onPress: (): boolean => navigate(ROUTES.PORTFOLIO_CARDS_OPERATIONS, { card: item }) } }
+          >
+            <Col size={8}>
+              <Text style={ [CreditCardStyle.textStyle, CreditCardStyle.smallTextStyle] }>
+                {item.lastUsage}
+              </Text>
+            </Col>
+            <Col size={1}>
+              <Icon
+                type = "MaterialIcons"
+                style={CreditCardStyle.iconStyle}
+                name="keyboard-arrow-right"
+              />
+            </Col>
+          </Row>
+        </Grid>
+      </Body>
+    </Card>
+    )
+  }
+
+  private getCreditCardImage: any = (creditCardNumber: string) => {
+
+    const creditCardType: CreditCardType =
+      CreditCard.getCardType(creditCardNumber)
+
+    switch (creditCardType) {
+      case CreditCardType.VISA:
+        return imgVisa
+      case CreditCardType.MASTERCARD:
+        return imgMastercard
+      case CreditCardType.DINERS:
+        return imgDiners
+      case CreditCardType.AMEX:
+        return imgAmex
+      case CreditCardType.MAESTRO:
+        return imgMaestro
+      case CreditCardType.VISAELECTRON:
+        return imgElectron
+      default:
+        return imgMastercard
```

Comment:
is this fallback the correct logic or it's temporary? in the latter case, create a pivotal story as reminder for fixing it